backend/gene_info/api/ncbi_provider.py

```diff
@@ -62,11 +62,15 @@ def fetch_gene_uid(self, geneID, gene):
             raise NCBIUnexpectedResultException
 
         # search with gene name if needed
-        if not self._is_valid_search_result(search_response) and gene != "":
+        if not self._is_valid_search_result(search_response) and gene and gene != "":
             try:
-                search_response = self._search_gene_uid(gene)
+                term = "(" + str(gene) + "%5BGene%20Name%5D)%20AND%20human%5BOrganism%5D"
+                search_response = self._search_gene_uid(term)
                 if self._is_valid_search_result(search_response):
-                    return (int(search_response["esearchresult"]["idlist"][0]), False)
+                    # switched is_ensembl_id_result to True with new search term
```

Comment:
this comment wont make much sense outside of the pr review (but I do appreciate it)

could you also add a comment to explaining the string on line 67 (or pull it out into a const with a clear name)?